contrib/trivy/parser/v2/parser.go

```diff
@@ -35,34 +35,21 @@ func (p ParserV2) Parse(vulnJSON []byte) (result *models.ScanResult, err error)
 }
 
 func setScanResultMeta(scanResult *models.ScanResult, report *types.Report) error {
-	const trivyTarget = "trivy-target"
-	for _, r := range report.Results {
-		if pkg.IsTrivySupportedOS(r.Type) {
-			scanResult.Family = r.Type
-			scanResult.ServerName = r.Target
-			scanResult.Optional = map[string]interface{}{
-				trivyTarget: r.Target,
-			}
-		} else if pkg.IsTrivySupportedLib(r.Type) {
-			if scanResult.Family == "" {
-				scanResult.Family = constant.ServerTypePseudo
-			}
-			if scanResult.ServerName == "" {
-				scanResult.ServerName = "library scan by trivy"
-			}
-			if _, ok := scanResult.Optional[trivyTarget]; !ok {
-				scanResult.Optional = map[string]interface{}{
-					trivyTarget: r.Target,
-				}
-			}
-		}
-		scanResult.ScannedAt = time.Now()
-		scanResult.ScannedBy = "trivy"
-		scanResult.ScannedVia = "trivy"
+	if len(report.Results) == 0 {
+		return xerrors.Errorf("scanned images or libraries are not supported by Trivy. see https://aquasecurity.github.io/trivy/dev/vulnerability/detection/os/, https://aquasecurity.github.io/trivy/dev/vulnerability/detection/language/")
 	}
 
-	if _, ok := scanResult.Optional[trivyTarget]; !ok {
-		return xerrors.Errorf("scanned images or libraries are not supported by Trivy. see https://aquasecurity.github.io/trivy/dev/vulnerability/detection/os/, https://aquasecurity.github.io/trivy/dev/vulnerability/detection/language/")
+	scanResult.ServerName = report.ArtifactName
```

Comment:
ArtifactName has iamge name and tag info.
(When scanned by Trivy without :latest tag, also omitted tag info from ArtifactName)